twisted/conch/ssh/keys.py

```diff
@@ -39,6 +42,47 @@
 from twisted.python.deprecate import deprecated, getDeprecationWarningString
 from twisted.python.versions import Version
 
+# Curve lookup table
+curveTable = {
```

Comment:
`curveTable` and `oidTable` are new public APIs according to Twisted's compatibility policy.  Do we actually need to expose these to applications?  I suspect not.